### ISSUE TITLE
Youtube "Scrobble only music category" never work

### ISSUE THREAD

**Issue by VladimirAmiorkov** [david-sabata/web-scrobbler#615]
I noticed that the Youtube "Scrobble only music category" never works for me. For example this video (https://www.youtube.com/watch?v=Mb1ZvUDvLDY&feature=youtu.be) should scrobble but the web scrobblers icons simply stays saying "This site is supported fro scrobbling"

**Comment by aemxdp** [comment]
This is #590 again. Fixed in https://github.com/david-sabata/web-scrobbler/pull/594.

**Comment by VladimirAmiorkov** [comment]
If this was fixed why I am still unable to scrollble from Youtube when the "Scrobble only music category" is checked. If I removed that setting and simply make Youtube scrobble everything, it works but that is not what I want.

Is it possible I am missing something about the "Scrobble only music category" feature. What is considered music category and how can I be sure I am looking at a music video that is from that category. For example every VEVO clip should be in that category right, but even those do not scrobble. I reinstalled the extension and its version is 1.47. Not sure what else I could do tbh.
